### ISSUE TITLE
Reduce log output from WebDAV

### ISSUE THREAD

**Issue by carlolars** [danmunn/redmine_dmsf#588]
We have a lot of projects so instead of mounting each project as a network share I mount dmsf/webdav/.
The problem is that Windows is very serious about keeping the folder properties up to date, so it makes PROPFIND requests for all subfolders (projects) every 3-5 second. This makes the production.log unusable for anything other than knowing that windows is busy...
Is there a way to decrease the amount of output from WebDAV without decreasing the log level for the entire redmine system?
Is it possible to redirect the WebDAV logging to another log-file?

**Comment by picman** [comment]
Check the config/routes.rb an modify _:log_to_ parameter:
```ruby
# DAV4Rack implementation of Webdav
  mount DAV4Rack::Handler.new(
    :root_uri_path => "#{Redmine::Utils::relative_url_root}/dmsf/webdav",
    :resource_class => RedmineDmsf::Webdav::ResourceProxy,
    :controller_class => RedmineDmsf::Webdav::Controller,
    :log_to => Rails.logger
  ), :at => '/dmsf/webdav'  if defined?(RedmineDmsf)
```

**Comment by eurodba** [comment]
Nice to know!

Maybe it matters to add additional option in plugin's Settings page - so that everyone can configure detalisation of WebDAV's activity into production.log. Or even whole DMSF activity (like "normal", "debug" etc.).

And funny is that even Windows makes so much requests for files/folders timestamps - it's well-known that for WebDAV files/folders timestamps are unuseful, e.g. they are always incorrect and shows some strange values. And that is BIG problem for using DMSF via WebDAV. But that is another story..


Rostislav

**Comment by picman** [comment]
Concerning invalid timestamps, please see recently closed issue #581.
